File: java/UKL/soal2.java
/*
Soal    : 2
A       : 5
B       : 6
*/

public class soal2 {
   public static void main(String[] args) {
        int A=5,
            B=9, 
            u=A, 
            s=A; 
        System.out.println("Deret Aritmatikanya adalah"); 
        for(int i=0;i<5;i++){                            
          for(int j=0;j<6;j++){  
            System.out.print(u+"\t"); 
            u=u+B;
            s=s+u;
          }
          System.out.println(); 
        }
        s=s-u;
        System.out.println("Jumlah Deret Aritmatikanya adalah "+s); 
    }
}

File: golang/UKL/soal2.go
/*
Soal    : 2
A       : 5
B       : 6
*/

package main

import "fmt"

func main() {
	A := 5
	B := 9
	u := A
	s := A
	fmt.Println("Deret Aritmatikanya adalah")
	for i := 0; i < 5; i++ {
		for j := 0; j < 6; j++ {
			fmt.Print(u, "\t")
			u = u + B
			s = s + u
		}
		fmt.Println()
	}
	s = s - u
	fmt.Println("Jumlah Deret Aritmatikanya adalah ", s)
}
